LCD_digits/digits/DigitPatterns.js

```diff
@@ -0,0 +1,89 @@
+const ZERO = [  
+    " _ ",
+    "| |",
+    "|_|"
+];
+
+const ONE = [
+    "   ",
+    "  |",
+    "  |"
+];
+
+const TWO = [  
+    " _ ",
+    " _|",
+    "|_ "
+];
+
+const THREE = [
+    " _ ",
+    " _|",
+    " _|"
+];
+
+const FOUR = [
+    "   ",
+    "|_|",
+    "  |"
+];
+
+const FIVE = [
+    " _ ",
+    "|_ ",
+    " _|"
+];
+
+const SIX =  [
+    " _ ",
+    "|_ ",
+    "|_|"
+];
+
+const SEVEN = [
+    " _ ",
+    "  |",
+    "  |"
+];
+
+const EIGHT = [
+    " _ ",
+    "|_|",
+    "|_|"
+];
+
+const NINE = [
+    " _ ",
+    "|_|",
+    "  |"
+];
+
+function getDigitPattern(digit) {
```

Comment:
You could just use the number as the key to the object. Either a string or putting it in brackets

```javascript
const Patterns = {
    "0": [
        " _ ",
        "| |",
        "|_|"
      ],
// ...
[5]: [
        " _ ",
        "|_ ",
        " _|"
    ],

    [6]:  [
        " _ ",
        "|_ ",
        "|_|"
    ],
// ...
};
```

and then you could simplify the method and remove the switch

```javascript
function getDigitPattern(digit) {
    const pattern = Patterns[digit];
    if (!pattern) throw new Error("Expected single digit 0-9");
    return pattern;
}
```